extension/css/popup.css

```diff
@@ -8,9 +8,11 @@
   background-position: top center;
   width: var(--panelWidth);
   height: var(--panelHeight);
```

Comment:
Q: Should `--panelWidth` and `--panelHeight` be prefixed here also? Not sure the likelihood of collisions there.